### DISCUSSION TITLE
Alpine.js version

### DISCUSSION THREAD

**Discussion by nhedger** [livewire/livewire#7826] [1. Help] [answered]
Hi,

While reading the [**Manually bundling Livewire and Alpine**](https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine) section of the installation instructions, I found the following snippet.

```ts
import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm';
import Clipboard from '@ryangjchandler/alpine-clipboard'
 
Alpine.plugin(Clipboard)
 
Livewire.start()
```

As this snippet suggests, Alpine is bundled with Livewire. 

I'm assuming this means that Livewire may not necessarily have the _latest_ version of Alpine.js at all times. 

I guess I'm asking whether loading Alpine from `node_modules` would be an acceptable alternative. I haven't encountered any errors yet, but I just wanted to make sure I'm not missing something.

```ts
import Alpine from 'alpinejs'
import { Livewire } from '../../vendor/livewire/livewire/dist/livewire.esm'
import Clipboard from '@ryangjchandler/alpine-clipboard'
 
Alpine.plugin(Clipboard)

Alpine.start()
 
Livewire.start()
```

**Comment by joshhanley** [comment, marked as answer]
@nhedger I wouldn't recommend it, I would follow it as we have documented, as under the hood we register things that Livewire needs to run.

We keep Alpine up to date inside Livewire so you don't need to worry about it. To the point that the version of Alpine that Livewire has bundled with generally has fixes included that may not have been released in a tagged Alpine version yet (so won't yet be available on NPM).

**Reply by nhedger** [reply]
Got it, thanks for your time!

**Reply by joshhanley** [reply]
@nhedger no worries! 🙂
